DEFAULT_CONFIG.json5

```diff
@@ -122,7 +122,19 @@
       //     cancel_goal: 1.0,
       //     get_result: [".*long_mission=3600", ".*short_action=10.0"],
       //   }
-      // }
+      // },
+
+      ////
+      //// work_thread_num: The number of worker thread in TOKIO runtime (default: 2)
+      ////                  The configuration only takes effect if running as a dynamic plugin, which can not reuse the current runtime.
```

Comment:
Suggestion:
"... if running as a dynamic plugin **(i.e. not as a standalone bridge)**, ..."

Reply:
Done.